src/pages/subscriptions.js

```diff
@@ -19,28 +18,22 @@ class SubscriptionsPage extends React.Component {
 
   static propTypes = {
     slug: PropTypes.string,
-    getLoggedInUser: PropTypes.func.isRequired, // from withLoggedInUser
+    LoggedInUser: PropTypes.func.isRequired, // from withLoggedInUser
   };
 
   constructor(props) {
     super(props);
     this.state = {};
```

Comment:
If the state is not used anymore we can remove it (as well as the `constructor`)

Reply:
Done.